packages/now-cli/src/util/ignored.ts

```diff
@@ -1,25 +0,0 @@
-// Base `.gitignore` to which we add entries
-// supplied by the user
-export default `.hg
-.git
-.gitmodules
-.svn
-.cache
-.next
-.now
-.npmignore
-.dockerignore
-.gitignore
-.*.swp
-.DS_Store
-.wafpicke-*
-.lock-wscript
-.env
-.env.*
-.venv
-npm-debug.log
-config.gypi
-node_modules
-__pycache__/
-venv/
-CVS`;
```

Comment:
This is great because we had a duplicate it `now-client` so now there is one source of truth! 🎉

Reply:
Exactly, there is probably even more that can be unified, e.g. the getFiles function that is used in `now-dev`